proxmoxtf/resource/vm.go

```diff
@@ -1600,7 +1600,44 @@ func deleteIdeDrives(ctx context.Context, vmAPI *vms.Client, itf1 string, itf2 s
 	return nil
 }
 
-// Shutdown the VM.
+// Wait until a VM reaches the desired state. An error is returned if the VM doesn't reach the
+// desired state within the given timeout.
+func vmWaitState(ctx context.Context, vmAPI *vms.Client, state string, timeout int) diag.Diagnostics {
```

Comment:
We can probably reuse this one? https://github.com/bpg/terraform-provider-proxmox/blob/9fd9d211d75e760ef1c7e44d13de9ce8d38bf834/proxmox/nodes/vms/vms.go#L549

Reply:
Oh! I hadn't seen it. Thanks for catching that.